### DISCUSSION TITLE
Trails in the sky

### DISCUSSION THREAD

**Discussion by iamacookie** [jserodio/rg351p-ppsspp-settings#18] [Q&A - Drop your questions here]
Hello! I’ve recently stumbled into your community and it’s pretty awesome!
Let me be candid, I am very new to emulation, as i’m currently shopping for my first handheld emu device. I want to get the rg351 but I need to know if trails in the sky works on it!
So I’m not here so much to contribute, more to request if anyone can test it for me before I make my purchase. 

I will however be testing other jrpgs after and would love to contribute any way I can if trails in the sky works out. (Otherwise I’ll be getting the retroid pocket 2 which I know can play it).
I hope my request isn’t inappropriate, I’m a bit new to this world, and I have a lot of respect for the work you guys have done on this list.
Thank you, Charlie.

**Comment by jserodio** [comment]
Hey man, I hope I'm not late before you make the mistake on buying a retroid pocket 2 😅 (like I did)
I currently these devices own 
Rg350
Rg351p
Rg351m
Retroid pocket 2
And my last acquisition rgb10 max panda edition

Basically 351 series and rgb10 max use the same chip and have the advantage of loading SD cards with everything configured by the community. You just can't do this on the retroid pocket 2, so you are stuck with what you are given. Android or retroid OS, and no unique interface to use.

Apart from obvious differences on hardware, I'm gonna try this game for you before you make any purchases.
I'll try to be quick, give me some hours

**Comment by jserodio** [comment]
Hey man, good news, I played until chapter 2 and everything is smooth and fine without touching any settings.
This game will work perfectly on any of these devices:
Rg351P
Rg351M
RGB10 MAX

**Reply by iamacookie** [reply]
Thank you so much for your time! I'm very pleased to hear it works well, I'll be getting the 351m ordered when I get home. I really appreciate your help, I can start getting excited now!
Thank you,
Charlie
